src/app/listing-view/listing-view.component.ts

```diff
@@ -0,0 +1,27 @@
+import {Component, OnInit} from '@angular/core';
+import {Listing} from '../../model/listing';
+import {ListingService} from '../../services/listing.service';
+import {Router} from '@angular/router';
+
+@Component({
+  selector: 'app-listing-view',
+  templateUrl: './listing-view.component.html',
+  styleUrls: ['./listing-view.component.css'],
+})
+export class ListingViewComponent implements OnInit {
+  listings: Listing[];
+  listing1: Listing;
```

Comment:
I don't like `listing1` as a variable name. Does `listing` not work? If not, maybe we could change the name slightly so we don't need to tack on a number at the end?

Reply:
yeah forgot to change that, it was there for debugging. wont even need the var with the new code though